base/test/sound_record_tests.cpp

```diff
@@ -18,18 +18,18 @@ BOOST_AUTO_TEST_SUITE(sound_record_tests)
 
 BOOST_AUTO_TEST_CASE(recordMono, *boost::unit_test::disabled())
 {
-	std::vector<std::string> audioFiles = { "./data/AudiotestMono.wav" };
-	Test_Utils::FileCleaner f(audioFiles);
+    std::vector<std::string> audioFiles = { "./data/AudiotestMono.wav" };
+    Test_Utils::FileCleaner f(audioFiles);
     // Manual test, listen to the file on audacity to for sanity check
     Logger::setLogLevel(boost::log::trivial::severity_level::info);
 
     auto time_to_run = Test_Utils::getArgValue("s", "10");
     auto n_seconds = atoi(time_to_run.c_str());
-    auto sampling_rate = 48000;
+    auto sampling_rate = 16000;
```

Comment:
curious why this had to be changed in this test ?

Reply:
Not needed. Will revert.

Reply:
Done